projects/go-lib/src/styles/_variables.scss

```diff
@@ -9,7 +9,7 @@ $global-transition-duration: .25s;
 
 // Box Shadows
 $global-box-shadow--regular: 0 0 2px 0 rgba(0, 0, 0, 0.12), 0 3px 6px 0 rgba(0, 0, 0, 0.20);
-$global-box-shadow--search: 0 3px 6px rgba(0, 0, 0, .2); // TODO: verify with UX proper box-shadow for go-search
+$global-box-shadow--search: 0px 1px 6px 0px rgba(32,33,36,0.28);
```

Comment:
Would you mind cleaning up the formatting of this declaration a little bit please?

```
$global-box-shadow--search: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
```

Reply:
Done.